services/brig/src/Brig/API/User.hs

```diff
@@ -359,7 +359,7 @@ createUserInviteViaScim (NewUserScimInvitation uid tid loc name rawEmail) = (`ca
   account <- lift $ newAccountInviteViaScim uid tid loc name email
   Log.debug $ field "user" (toByteString . userId . accountUser $ account) . field "action" (Log.val "User.createUserInviteViaScim")
   ttl <- round . setTeamInvitationTimeout <$> view settings
-  lift $ Data.insertAccount account Nothing Nothing False (Just ttl)
+  lift $ Data.insertAccount account Nothing Nothing True (Just ttl)
```

Comment:
no.

activated flag has to be false, otherwise we are risking to see it in all kinds of places where we shouldn't.  what does "otherwise on id" mean?